errors/error.go

```diff
@@ -27,12 +27,27 @@ const (
 	// DBOpen indicates a database open error.
 	DBOpen = ErrorKind("ErrDBOpen")
 
+	// DBOpen indicates a database close error.
+	DBClose = ErrorKind("ErrDBClose")
+
 	// DBUpgrade indicates a database upgrade error.
 	DBUpgrade = ErrorKind("ErrDBUpgrade")
 
 	// PersistEntry indicates a database persistence error.
 	PersistEntry = ErrorKind("ErrPersistEntry")
 
+	// CreateTable indicates a database table creation error.
+	CreateTable = ErrorKind("CreateTable")
+
+	// BeginTx indicates a database transaction begin error.
+	BeginTx = ErrorKind("BeginTx")
```

Comment:
All of these tx errors can probably just be `ErrPersistEntry`. Right now, the calling doesn't need to know the precise nature of the failure, its enough just to know that something went wrong. The nested error generated by the SQL library will contain more details for the log file.